src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -78,7 +82,7 @@ export const DropDownNavMenu: React.FC<DropDownNavMenuProps> = ({
                 const isLast = menu.links.length - 1 === i
 
                 return isLast ? (
-                  <ViewAllMenuItem key={menuItem.text}>
+                  <ViewAllMenuItem key={menuItem.text} href={menuItem.href}>
```

Comment:
This is a `RouterLink` underneath, yah?

Reply:
Yeah: https://github.com/artsy/force/blob/master/src/v2/Components/Menu/MenuItem.tsx

Reply:
👌

Reply:
Does `RouterLink` put href into to? It doesn't appear so. (I basically just tried to simplify what was here so I wasn't looking at any of the actual implementation)